tests/test_grids.py

```diff
@@ -343,6 +343,9 @@ def test_nemo_grid(mode):
     dimensions = {'lon': 'glamf', 'lat': 'gphif'}
     field_set = FieldSet.from_nemo(filenames, variables, dimensions)
 
+    # test ParticleSet.from_field on curvilinear grids
+    ParticleSet.from_field(field_set, ptype[mode], start_field=field_set.U, size=5)
+
```

Comment:
Does this still need an assert statement to check if the particles are indeed roughly homogenous? Because the `ParticleSet.from_field()` statement above was simply to check if there was no error, for now

Reply:
Done in test_particle_sets.py